### PR TITLE
fix: New or changed files are parsed only when they are opened in the editor

### PR DESCRIPTION
**Pull request by jirimosinger**

### CODE REVIEW

File: clients/vscode-hlasmplugin/.vscode/launch.json [hunk 1]
```diff
@@ -1,30 +1,36 @@
...
         },
         {
             "name": "Launch Extension",
             "type": "extensionHost",
             "request": "launch",
             "runtimeExecutable": "${execPath}",
-            "args": ["--extensionDevelopmentPath=${workspaceRoot}" ],
-            "stopOnEntry": false,
+            "args": [
+                "--extensionDevelopmentPath=${workspaceFolder}"
```
Comment (slavek-kucera) [inline clients/vscode-hlasmplugin/.vscode/launch.json:28]: The stopOnEntry has not effect here?

**Comment by jirimosinger** [comment]
Exactly. 

Probably one of the worst situations is when download dependencies command is used, the extension starts to process every single download file

**Review by slavek-kucera — APPROVED** [review]
👍

File: clients/vscode-hlasmplugin/.vscode/launch.json (hunk 1 above)
Comment (jirimosinger) [reply clients/vscode-hlasmplugin/.vscode/launch.json:28]: Nope, not in this case. It even says the following:
image

**Pull request merged**